main.go

```diff
@@ -143,27 +156,7 @@ func destroyResources(req *superkey.DestroyRequest) {
 	l.Log.Infof("Finished Un-Forging request: %v", req)
 }
 
-func parseSuperKeyCreateRequest(value []byte) (*superkey.CreateRequest, error) {
-	request := superkey.CreateRequest{}
-	err := json.Unmarshal(value, &request)
-	if err != nil {
-		return nil, err
-	}
-
-	return &request, nil
-}
-
-func parseSuperKeyDestroyRequest(value []byte) (*superkey.DestroyRequest, error) {
-	request := superkey.DestroyRequest{}
-	err := json.Unmarshal(value, &request)
-	if err != nil {
-		return nil, err
-	}
-
-	return &request, nil
-}
-
-func getHeader(name string, headers []kafka.Header) string {
+func getHeader(name string, headers []protocol.Header) string {
```

Comment:
It looks that there possibility to remove dependency on  `github.com/segmentio/kafka-go/protocol`. Maybe we can move method `getHeader` to `github.com/RedHatInsights/sources-api-go/kafka`.

Reply:
Yeah I like that idea - its a piece of code we use commonly from kafka consumers so it would make sense to have it as an exported function there. I'll open that PR when I get time.

Reply:
I ran `go get -u github.com/RedHatInsights/sources-api-go/kafka` and updated the implementation to use the new method. 

Was able to remove a lot of functions from this app!

Reply:
great, thanks 👍